logic/path/src/selectors/legacy.rs

```diff
@@ -6,14 +6,19 @@ use hopr_crypto_random::random_float;
 use hopr_internal_types::channels::ChannelEntry;
 use hopr_internal_types::protocol::INTERMEDIATE_HOPS;
 use hopr_primitive_types::prelude::*;
+use hopr_primitive_types::traits::UnitaryFloatOps;
```

Comment:
Not needed, because it should be already imported via `prelude` on the previous line